File: Data Structures & Algorithms/two-integer-sum/submission-0.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        # {"value": "index"}
        checked_nums: dict = {}

        for i in range(len(nums)):
            complement: int = target - nums[i]

            if complement in checked_nums:
                return [checked_nums[complement], i]
            
            checked_nums[nums[i]] = i

File: Data Structures & Algorithms/concatenation-of-array/submission-0.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        
        int[] extendedNums = new int[nums.length * 2];

        for(int i = 0; i < nums.length; i++){
            extendedNums[i] = nums[i];
            extendedNums[i + nums.length] = nums[i];
        }
        

        return extendedNums;
    }
}